google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -99,11 +116,6 @@ public OSProcessHandler createProcessHandler(
             executor.run();
 
             Process devappserverProcess = executor.getProcess();
-            if (devappserverProcess == null) {
```

Comment:
I'd prefer for shit to blow up in this case. This is truely an exception condition and it would be nice to be notified (via an error report) rather than just swallowing it with a UI message. Thoughts?